Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_TOOL = xcodebuild
-BUILD_SCHEME = SQLite Mac
```

Comment:
it should work via make's conditional =? operator (or ?=, can't remember the order)


Il giorno 8 luglio 2016 @ 16:16:27, Nick Shelley (notifications@github.com(mailto:notifications@github.com)) ha scritto:

>  
>  
> In Makefile(https://github.com/stephencelis/SQLite.swift/pull/459#discussion_r70080494):
>  
> > @@ -1,6 +1,9 @@ > BUILD_TOOL = xcodebuild > -BUILD_SCHEME = SQLite Mac  
>  
> I think it would need to be wrapped in an if [ -z "$BUILD_SCHEME" ] to make it act like a default instead of an override.
>  
>  
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub(https://github.com/stephencelis/SQLite.swift/pull/459/files/6759aad915b5bfc39f9b6f733785690ab69ef5e0#r70080494), or mute the thread(https://github.com/notifications/unsubscribe/AABJyzkQU3_BUhfRAX1DkGS6KYFU6mHCks5qTlu7gaJpZM4I9PdX).
>  
>  
>  
>  
>